lib/commands/hasFocus.js

```diff
@@ -27,19 +27,27 @@
  */
 
 let hasFocus = function (selector) {
+    const finalSelector = selector === null ? this.lastPromise.inspect().value.selector : selector
     let result = this.execute(function (selector) {
         var focused = document.activeElement
-
         if (!focused || focused === document.body) {
             return false
-        } else if (document.querySelector) {
-            return focused === document.querySelector(selector)
+        } else if (document.querySelectorAll) {
```

Comment:
Sorry Marek and I are scratching our heads, what do you mean wdio test runner test?